src/reducers/posts.js

```diff
@@ -0,0 +1,27 @@
+const initalState = ({
+    posts : [],
+    currentPostComments: [],
+    currentPost: "",
```

Comment:
Esse é o id do post atual ou o post atual de fato? Se for o post atual de fato, que é o que parece, acho que faz sentido que ele inicie como um objeto vazio ou undefined, ao invés de uma string vazia.